src/SessionManager/ProgramSettings.cc

```diff
@@ -54,7 +54,8 @@ ProgramSettings::ProgramSettings(int argc, char** argv) {
         ("frontend_folder", "Set folder to serve frontend files from", cxxopts::value<string>(), "<path>")
         ("exit_after", "Number of seconds to stay alive after last sessions exists", cxxopts::value<int>(), "<duration>")
         ("init_exit_after", "Number of seconds to stay alive at start if no clients connect", cxxopts::value<int>(), "<duration>")
-        ("files", "Files to load", cxxopts::value<vector<string>>(positional_arguments));
+        ("files", "Files to load", cxxopts::value<vector<string>>(positional_arguments))
+        ("idle_session_timeout", "Delete the session if it idles for the number of seconds", cxxopts::value<int>()->default_value(to_string(idle_session_timeout)), "<duration>");
```

Comment:
I don't think printing a default of -1 adds any information for the user. What does -1 mean? I suggest we follow the same approach as `exit_after` `init_exit_after` (using `applyOptionalArgument`).